groups.go

```diff
@@ -90,6 +95,16 @@ type Group struct {
 
 	// Deprecated: Use EmailsEnabled instead
 	EmailsDisabled bool `json:"emails_disabled"`
+
+	// Deprecated: Use DefaultBranchProtectionDefaults instead
+	DefaultBranchProtection int `json:"default_branch_protection"`
+}
+
+// GroupAccessLevel represents an entry in a Group defining the access level for default protections
```

Comment:
```suggestion
// GroupAccessLevel represents default branch protection defaults access levels.
```
Still not great, but close enough.